install/error-handling.sh

```diff
@@ -4,20 +4,32 @@ export SENTRY_DSN='https://19555c489ded4769978daae92f2346ca@self-hosted.getsentr
 export SENTRY_ORG=self-hosted
 export SENTRY_PROJECT=installer
 
+jq="docker run --rm -i mbently/jq"
+sentry_cli="docker run --rm -v /tmp:/work -e SENTRY_ORG=$SENTRY_ORG -e SENTRY_PROJECT=$SENTRY_PROJECT -e SENTRY_DSN=$SENTRY_DSN getsentry/sentry-cli"
+
 send_envelope() {
   # Send envelope
-  local sentry_cli="docker run --rm -v /tmp:/work -e SENTRY_ORG=$SENTRY_ORG -e SENTRY_PROJECT=$SENTRY_PROJECT -e SENTRY_DSN=$SENTRY_DSN getsentry/sentry-cli"
-  $sentry_cli send-envelope $envelope_file
+  $sentry_cli send-envelope "$envelope_file"
+}
+
+generate_breadcrumb_json() {
+  # Based on https://stackoverflow.com/a/1521498
+  while IFS="" read -r line || [ -n "$line" ]; do
+    jq -n -c --arg message "$line" \
+      --arg category log \
+      --arg level info \
+      '$ARGS.named'
+  done <$log_path
 }
 
 send_event() {
   # Use traceback hash as the UUID since it is 32 characters long
   local event_hash=$1
   local traceback=$2
```

Comment:
I'm going to change this before merge as I'd like to rename this to "error" or something, because it isn't actually the traceback.

Reply:
just wanted to make sure this was done! since everything lgtm

Reply:
Good catch, pushed a change for this.